### PR TITLE
refactor(ui): creating kube actions component

### PR DESCRIPTION
**Pull request by axel7083**
### What does this PR do?

https://github.com/podman-desktop/podman-desktop/pull/10073 will add a new actions to kube, which will be displayed in each page, we can simplify by creating a shared `KubeActions` component, to ease adding/removing actions in the future.

### Screenshot / video of UI

N/A

### What issues does this PR fix or reference?

Related to https://github.com/podman-desktop/podman-desktop/issues/8845

### How to test this PR?

- [x] Tests are covering the bug fix or the new feature

### CODE REVIEW

File: packages/renderer/src/lib/kube/KubeActions.svelte [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<script lang="ts">
+import KubeApplyYamlButton from '../kube/KubeApplyYAMLButton.svelte';
```
Comment (benoitf) [inline packages/renderer/src/lib/kube/KubeActions.svelte]: suggestion (non-blocking): here you could use the `/@/` import scheme

**Review by benoitf — APPROVED** [review]
thanks for the refactoring, it'll be easier to introduce new stuff without multiple copy/paste, forgetting one item, etc

File: packages/renderer/src/lib/kube/KubeActions.svelte [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+</script>
+
+<div class="flex flex-row gap-x-2">
```
Comment (deboer-tim) [inline packages/renderer/src/lib/kube/KubeActions.svelte]: The additional-action section already defines button spacing, we shouldn't create a subcomponent that applies its own.

Comment (axel7083) [reply packages/renderer/src/lib/kube/KubeActions.svelte]: You are right, I was not aware, fixed :+1:

File: packages/renderer/src/lib/kube/KubeActions.svelte (hunk 1 above)
Comment (axel7083) [reply packages/renderer/src/lib/kube/KubeActions.svelte]: Fixed :+1:

**Pull request merged**